tests/comm/test_trtllm_allreduce_checkpoint.py

```
@@ -0,0 +1,122 @@
+        dist.destroy_process_group()
+
+
+@pytest.mark.parametrize("num_tokens,use_oneshot", [(1, True), (4, False)])
+@pytest.mark.skipif(
+    not torch.cuda.is_available() or torch.cuda.device_count() < 2,
+    reason="checkpointable TRT-LLM all-reduce requires two CUDA devices",
+)
+def test_graph_replay_after_symmetric_memory_remap(
+    num_tokens: int, use_oneshot: bool
+) -> None:
+    os.environ.setdefault("FLASHINFER_WORKSPACE_BASE", "/tmp/flashinfer-tests")
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/flashinfer-tests"

(S108)